cuda_pathfinder/cuda/pathfinder/README.md

```diff
@@ -0,0 +1,46 @@
+# `cuda.pathfinder` Module
+
+## Public API for loading NVIDIA Dynamic Libs
+
+* `cuda.pathfinder.SUPPORTED_NVIDIA_LIBNAMES` (`tuple[str]`)
+
+* `cuda.pathfinder.load_nvidia_dynamic_lib(libname: str) -> LoadedDL`
+
+* `cuda.pathfinder.LoadedDL`:
+    * `handle` (platform-specific type)
+    * `abs_path` (`str`)
```

Comment:
Should we consider returning a pathlib object here instead of a string?

Reply:
My vote: No. For simplicity.

Plain `str` paths are still ubiquitous, and I'd guess always will be.